src/component/ChairList/ChairList.js

```diff
@@ -0,0 +1,70 @@
+import {MdArrowBackIos} from 'react-icons/md';
+import {useParams, useNavigate} from 'react-router-dom';
+import styled from 'styled-components';
+
+import ChairCard from '../ChairCard/ChairCard';
+
+export default function ChairList({chairsData, toggleDescription, showDescription}) {
+  const {name} = useParams();
+  let navigate = useNavigate();
+
+  return (
+    <>
+      <HeadingContainer>
+        <button onClick={() => navigate('/')}>
+          <MdArrowBackIos size={40} />
+        </button>
+        <h1>{name} chairs</h1>
+      </HeadingContainer>
+      <ChairContainer>
+        {chairsData
+          .filter(item => name === item.style)
+          .map(item => (
+            <ChairCard
+              key={item._id}
+              id={item._id}
+              name={item.name}
+              mainImg={item.imgUrl}
+              designerURL={item.designerURL}
+              designer={item.designer}
+              description={item.description}
+              toggleDescription={() => toggleDescription(item._id)}
+              showDescription={showDescription}
+              inspirationImg1={item.imgUrlAlternative1}
+              inspirationImg2={item.imgUrlAlternative2}
+              inspirationImg3={item.imgUrlAlternative3}
+            />
+          ))}
```

Comment:
You can remove all of these then :)
```suggestion
export default function ChairList({chairsData}) {
  const {name} = useParams();
  let navigate = useNavigate();

  return (
    <>
      <HeadingContainer>
        <button onClick={() => navigate('/')}>
          <MdArrowBackIos size={40} />
        </button>
        <h1>{name} chairs</h1>
      </HeadingContainer>
      <ChairContainer>
        {chairsData
          .filter(item => name === item.style)
          .map(item => (
            <ChairCard
              key={item._id}
              id={item._id}
              name={item.name}
              mainImg={item.imgUrl}
              designerURL={item.designerURL}
              designer={item.designer}
              description={item.description}
              inspirationImg1={item.imgUrlAlternative1}
              inspirationImg2={item.imgUrlAlternative2}
              inspirationImg3={item.imgUrlAlternative3}
            />
          ))}
```